game/scripts/vscripts/modifiers/modifier_octarine_vampirism_buff.lua

```diff
@@ -2,103 +2,117 @@ modifier_octarine_vampirism_buff = class(ModifierBaseClass)
 
 --------------------------------------------------------------------------------
 function modifier_octarine_vampirism_buff:DeclareFunctions(params)
-local funcs = {
-  MODIFIER_EVENT_ON_TAKEDAMAGE,
-  MODIFIER_PROPERTY_TOOLTIP
-  }
-  return funcs
-end
+	local funcs = {
+		MODIFIER_EVENT_ON_TAKEDAMAGE,
+		MODIFIER_PROPERTY_TOOLTIP
+	}
 
-function modifier_octarine_vampirism_buff:IsHidden()
-  return true
+	return funcs
 end
 
-function modifier_octarine_vampirism_buff:IsBuff()
-  if self:GetCaster():GetTeamNumber() == self:GetParent():GetTeamNumber() then
-    return true
-  end
-
-  return false
+function modifier_octarine_vampirism_buff:IsHidden()
+	return true
 end
 
 function modifier_octarine_vampirism_buff:IsPurgable()
-  return false
+	return false
```

Comment:
It looks like you accidentally messed up the indentation for the whole rest of this file. If you install that editorconfig plugin and re-save this file it should fix itself.